packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -34,7 +35,7 @@ export interface IButtonProps extends React.HTMLAttributes<HTMLButtonElement | H
   /**
    * Custom styling for individual elements within the button DOM.
    */
-  styles?: IButtonStyles;
+  styles?: ISplitButtonStyles;
```

Comment:
ahm this is weird? should I use styles even when the button is not a SplitButton?